lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -656,6 +657,26 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
           }
         end
 
+        parameters[:tax_amount] = options[:tax_amount] if options[:tax_amount]
+        parameters[:tax_exempt] = options[:tax_exempt] if options[:tax_exempt]
+        parameters[:purchase_order_number] = options[:purchase_order_number] if options[:purchase_order_number]
+
+        parameters[:shipping_amount] = options[:shipping_amount] if options[:shipping_amount]
+        parameters[:discount_amount] = options[:discount_amount] if options[:discount_amount]
+        parameters[:ships_from_postal_code] = options[:ships_from_postal_code] if options[:ships_from_postal_code]
+
```

Comment:
Are the fields `shipping.postal_code` and `shipping.country_code_alpha3` already handled elsewhere? I noticed those in the [docs ](https://developers.braintreepayments.com/reference/general/level-2-and-3-processing/required-fields/ruby#level-3-data) but don't see them built in right here.

Reply:
Yep, in https://github.com/activemerchant/active_merchant/blob/35666a09c074ebc2c7e09f9cfa2cb8d1fc4c9e70/lib/active_merchant/billing/gateways/braintree_blue.rb#L318